utilities.js

```diff
@@ -25,4 +25,10 @@ function formatDate(date) {
         minute: "2-digit",
         hour12: true
     });
+}
+
+function setAttributes(element, attributes) {
+    for (const attribute of attributes) {
+        element.setAttribute(attribute[0], attribute[1]);
```

Comment:
Here is my suggestion for better readability;

```suggestion
        const [key, value] = attribute;
        element.setAttribute(key, value);
```

Reply:
done